packages/material-ui/src/Popover/Popover.js

```diff
@@ -573,7 +578,7 @@ Popover.propTypes = {
   /**
    * Props applied to the [`Paper`](/api/paper/) element.
    */
-  PaperProps: PropTypes.shape({
+  PaperProps: PropTypes /* @typescript-to-proptypes-ignore */.shape({
```

Comment:
It may be possible to express this at the type level but this would reject our own components (React.ComponentType) that would work at runtime.